helix.go

```diff
@@ -118,9 +119,14 @@ func NewClient(options *Options) (*Client, error) {
 		options.HTTPClient = http.DefaultClient
 	}
 
+	baseURL := APIBaseURL
```

Comment:
Let's change the following lines this:

```go
if options.APIBaseURL == "" {
	options.APIBaseURL = APIBaseURL
}
```
That way we use the default value, if the `APIBaseURL` option is not provided.

Reply:
That's the way I wanted to do it at first, would look better in any case. I just thought it might be better not to modify the ``Options``-structure you get from a user to avoid unwanted side effects for the user. But I see now that you also have the ``HTTPClient'' field that you treat in a similar way.